src/task3/task1/PersonMain.java

```diff
@@ -0,0 +1,19 @@
+package task3.task1;
+
+import java.util.Random;
+
+public class PersonMain {
+    public static void main(String[] args) {
+        String[][] firstNameArray = {{"Михаил", "Мария", "Андрей", "Юлия", "Кристина"}, {"m", "f", "m", "f", "f"}};
+        String[] lastNameArray = {"Петров", "Сидоров", "Иванов", "Смирнов", "Кузнецов", "Попов"};
+        Person[] persons = new Person[50];
```

Comment:
it's magig number
Please use variables for anything numbers exclude 0 and 1

Reply:
Done